.projen/files.json

```diff
@@ -184,6 +184,7 @@
     ".github/workflows/release-trendmicro-cloudonecontainer-helm.yml",
     ".github/workflows/release-unxpose-iam-integration-module.yml",
     ".github/workflows/release-zmk-iam-lambdabasicrole-module.yml",
+    ".github/workflows/release.yml",
```

Comment:
🤔

Reply:
Since we are turning back the release for `cdk-cloudformation` this file is being added which was removed when we switched to manual release: https://github.com/cdklabs/cdk-cloudformation/pull/361/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34